lib/active_merchant/billing/gateways/priority.rb

```diff
@@ -0,0 +1,387 @@
+        params['bankAccount'] = nil
+      end
+
+      def add_bank_amount_refund(params, amount, authCode)
+        params['amount'] = amount
+        params['authCode'] = authCode
+        params['authMessage'] = ''
+        params['authOnly'] = false
+        params['availableAuthAmount'] = 0
+        params['batch'] = nil
+        params['batchId'] = nil
+      end
+
+      def add_credit_card(params, credit_card, action, options)
+        return unless credit_card
+
+        params['cardAccount'] ||= {}
+        card_details = params['cardAccount'] = {}
+
+        case action
+        when 'purchase'
+          card_details['avsStreet'] = options[:avsStreet]
+          card_details['avsZip'] = options[:avsZip]
+          card_details['cvv'] = credit_card.verification_value
+          card_details['entryMode'] = 'Keyed'
+          card_details['expiryDate'] = expdate(credit_card)
+          card_details['expiryMonth'] = format(credit_card.month, :two_digits).to_s
+          card_details['expiryYear'] = format(credit_card.year, :two_digits).to_s
+          card_details['last4'] = nil
+          card_details['magstripe'] = nil
+          card_details['number'] = credit_card.number
+        when 'refund'
+          card_details['cardId'] = credit_card['cardId']
+          card_details['cardPresent'] = credit_card['cardPresent']
+          card_details['cardType'] = credit_card['cardType']
+          card_details['entryMode'] = credit_card['entryMode']
+          card_details['expiryMonth'] = credit_card['expiryMonth']
+          card_details['expiryYear'] = credit_card['expiryYear']
+          card_details['hasContract'] = credit_card['hasContract']
+          card_details['isCorp'] = credit_card['isCorp']
+          card_details['isDebit'] = credit_card['isDebit']
+          card_details['last4'] = credit_card['last4']
+          card_details['token'] = credit_card['token']
+        end
+      end
+
+      def expdate(credit_card)
+        "#{format(credit_card.month, :two_digits)}/#{format(credit_card.year, :two_digits)}"
+      end
+
+      @request_params = {
+        achIndicator: nil,
+        authCode: nil,
+        authOnly: false,
+        bankAccount: nil,
+        cardPresent: false,
+        cardPresentType: 'CardNotPresent',
+        isAuth: true,
+        isSettleFunds: true,
+        isTicket: false,
+        merchantId: 514_391_592,
+        mxAdvantageEnabled: false,
+        mxAdvantageFeeLabel: '',
+        paymentType: 'Sale',
+        purchases: [{ taxRate: '0.0000', additionalTaxRate: nil, discountRate: nil }],
+        shouldGetCreditCardLevel: true,
+        shouldVaultCard: true,
+        source: 'QuickPay',
+        sourceZip: '94102',
+        taxExempt: false,
+        tenderType: 'Card',
+        terminals: []
+      }
+      def purchases
+        [{ taxRate: '0.0000', additionalTaxRate: nil, discountRate: nil }]
+      end
+
+      def add_type_merchant_purchase(params, merchant, isSettleFunds, options)
```

Comment:
`isSettleFunds` should be `is_settle_funds` to match Ruby convention

Reply:
Hey @mikebyrneprth ! 

The `isSettleFunds` is now showing `is_Settle_Funds` instead of `is_settle_funds`. Also, it looks like any of the areas where they've been changed to Ruby convention of _'s was perhaps find and replaced, which potentially will cause what's sent to Priority to fail, for example:

`        params['is_Settle_Funds'] = is_Settle_Funds` <- I assume Priority would need this to be `params[isSettleFunds]`